src/features/plans/pages/PlanDetailPage.tsx

```
@@ -0,0 +1,198 @@
+              </span>
+            </div>
+          </div>
+
+          {/* 관리 버튼 */}
+          <div className="relative">
+            <button
+              onClick={() => setIsManageOpen(prev => !prev)}
+              className="rounded-md border border-border/50 px-3 py-1.5 text-sm hover:bg-secondary"
+            >
+              관리
+            </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)